### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Azure Sign In: Unable to navigate to “Azure Sign In” button using Keyboard.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3310]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
OS Details: 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.1.1

**Repro Steps:** 
1) Open IntelliJ IDEA application
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application, and tab till “Azure Sign In “ button.

**Actual Result:**
Verify user is able to navigate to “ Azure Sign In” button using tab key.
Expected Result:
User should be able to navigate to “Azure Sign In “ button using tab key. Whenever focus is on “Azure Sign in” button tooltip should also be visible to users.
User Impact:
Keyboard users can not perform all operations if few controls are not accessible using keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Comment by Accessibility56** [comment]
![2 Azure Sign in](https://user-images.githubusercontent.com/38523250/62202841-b0491280-b3a7-11e9-9d8f-cf482e828627.png)

**Comment by Flanker32** [comment]
@Accessibility56 ActionButton & HideableDecorator cannot get focus using tab, tracking here https://youtrack.jetbrains.com/issue/IDEA-191322. And users could sign in with Tools/Azure/Azure Sign in, which could be done with keyboards only, so this issue should not bother users who use keyboard only.

**Comment by Flanker32** [comment]
Importance of experience: Important, users need to auth with toolkit first and then use the Azure services.
Quality of experience: not impacted. Toolkit provide another way for user to auth with Toolkits, which is auth from menu  Tools/Azure/Azure Sign in, this could be done with keyboard only.
Alternate solutions: No alternate technologies or options can be leveraged.
Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
![3310](https://user-images.githubusercontent.com/12445236/63918038-7a628100-ca6e-11e9-8aa0-ccf250a57838.gif)

**Reproduce**
- Press `Alt + F`
- Press Right until `Tools`
- Press Down until `Azure`
- Press Right
- Press Enter

**Comment by soumyatippapuram** [comment]
We are able to access "Azure Sign In" from Tools>Azure>Azure Sign in using keyboard.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
@soumyatippapuram We have link this issue to AAT [here](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it.

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749778
